packages/components/pages/f-loyalty/src/components/LoyaltyCards.vue

```diff
@@ -50,16 +49,36 @@ export default {
     data () {
         return {
             pushToDataLayer: () => {},
-            tags: 'loyalty'
+            tags: 'loyalty',
+            adapters: []
         };
     },
 
+
     computed: {
         ...mapState(VUEX_MODULE_NAMESPACE_LOYALTY, [
             'globalUserId',
-            'brazeApiKey'
+            'brazeApiKey',
+            'authToken',
+            'tenant',
+            'stampCardsAPIUrl',
+            'inStampCardsAdapterExperiment'
         ])
+    },
+
+    created () {
+        this.adapters.push(this.inStampCardsAdapterExperiment ? stampCardsAdapter({
```

Comment:
Would you mind changing this so it is formatted like the below, imo it reads a little clearer:
```javascript
this.adapters.push(this.inStampCardsAdapterExperiment
? stampCardsAdapter({...})
: brazeAdapter({})